<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileList</class>
 <widget class="QWidget" name="FileList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
